src/js/components/Pagination/__tests__/Pagination-test.tsx

```diff
@@ -399,4 +399,43 @@ describe('Pagination', () => {
     expect(getByLabelText('pagination-test-2')).toBeTruthy();
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('should apply a select component with default values for stepOptions', () => {
+    const { asFragment } = render(
+      <Grommet>
+        <Pagination numberItems={NUM_ITEMS} stepOptions />
+      </Grommet>,
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  test('should apply a select component with custom values for stepOptions', () => {
+    const { asFragment } = render(
+      <Grommet>
+        <Pagination
+          numberItems={NUM_ITEMS}
+          stepOptions={[10, 20, 30, 40, 50]}
+        />
+      </Grommet>,
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
Do we need to open the select drop for the snapshot to be effective for this test case?

Reply:
aw yes I'll add

Reply:
done